benchmark/conversion/conversion.cpp

```diff
@@ -118,6 +118,8 @@ struct ConversionBenchmark : Benchmark<gko::device_matrix_data<etype, itype>> {
     {
         gko::matrix_data<etype, itype> data;
         data = Generator::generate_matrix_data(test_case);
+        // no reordering here, as it doesn't impact conversions beyond
+        // dense-sparse conversions
```

Comment:
Now that we have neat classes, we could document this, and generally what the benchmark is doing, by documenting the main struct?
This is rather a comment for the future.

Reply:
Do you mean the individual benchmarks, or the benchmark runner?